### PR TITLE
Cs incident fixes

### PR DESCRIPTION
**Pull request by David-BMS**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by julieschwartz18** [comment]
Doc review completer for the playbook-CrowdStrike_Falcon_-_Get_Detections_by_Incident.yml file. @David-BMS - please confirm:
- line 210: The description for the **Set no detections found** task is "Set a value in context under the key you entered." - can the description maybe be more specific? Something like "Sets the <put here the key name> key value in context to no detections found."
- line 263 - the type for CrowdStrike.FoundDetections output is string, should it be boolean? Is the output yes/no or true/false?

**Comment by julieschwartz18** [comment]
Doc review complete for the playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml file. @David-BMS - please confirm the following:
- line 283: the task name is "Map behaviors data to the context" - can it be changed to "Map behavioral data to the context"?
- line 379, 455: the task name is "Set Endpoint information to layout" - can it be changed to "Set endpoint information to layout" (endpoint not capitalized)?
- line 491, 564: the task name is "Set Process Details To Layout" - can it be changed to "Set process details to layout"?
- line 646: the task name is "Extract Indicators" - can it be changed to "Extract indicators"?
- line 775: CrowdStrike.FoundDetections type is string, should it be boolean?

**Comment by David-BMS** [comment]
> Doc review complete for the playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml file. @David-BMS - please confirm the following:
> 





@julieschwartz18 
> * line 283: the task name is "Map behaviors data to the context" - can it be changed to "Map behavioral data to the context"?
This is CrowdStrike terminology (behaviors) so wouldn't change the grammar here

> * line 379, 455: the task name is "Set Endpoint information to layout" - can it be changed to "Set endpoint information to layout" (endpoint not capitalized)?
Yes Fixed

> * line 491, 564: the task name is "Set Process Details To Layout" - can it be changed to "Set process details to layout"?
Yes Fixed

> * line 646: the task name is "Extract Indicators" - can it be changed to "Extract indicators"?
Yes Fixed

> * line 775: CrowdStrike.FoundDetections type is string, should it be boolean?
Booleans dont pass well as playbook outputs, so left it as a string

File: Packs/CrowdStrikeFalcon/Classifiers/classifier-CrowdStrike_Falcon_Mapper_6.5.json [hunk 1]
```diff
@@ -662,10 +662,139 @@
...
                     "simple": "behaviors.user_name"
                 },
-                "severity": {
-                    "simple": "fine_score"
-                }
-            }
+				"name": {
+					"complex": {
+						"filters": [],
+						"root": "incident_type",
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Classifiers/classifier-CrowdStrike_Falcon_Mapper_6.5.json:668]: There is no other way to properly perform this without appending incident type with hostnames and mitre info?

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml [hunk 2]
```diff
@@ -661,7 +625,159 @@ tasks:
...
+    - label: yes
+      condition:
+      - - operator: isNotEqualString
+          left:
+            value:
+              simple: CrowdStrike.FoundDetections
+            iscontext: true
+          right:
+            value:
+              simple: 'False'
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml:666]: Maybe we should add here also ignore case flag (to support both lower and upper case - even if we are talking on a key that your setting)

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Investigation_and_Response.yml [hunk 3]
```diff
@@ -1813,17 +1795,155 @@ tasks:
...
+      description: ''
+    nexttasks:
+      '#default#':
+      - '113'
+      yes:
+      - '130'
+    separatecontext: false
+    conditions:
+    - label: yes
+      condition:
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Investigation_and_Response.yml:1878]: Same here - supporting lower and upper case

**Review by ssokolovich — COMMENTED** [review]
1. Should move the Incident handling items ( True/False positive classification) in the `CrowdStrike Falcon Malware - Investigation and Response`->  to the right so the manual item will be properly adjusted.
2. Why there is hostname info in the context but we don’t have any of that in the layout? Mapper issue?  Same for description and etc (based on the incident that I checked in your Instance)
3. CrowdStrike Falcon Malware - Incident Enrichment - should be changed to be shared with context globally? If not, add ExtractedIndicators to the output of this playbook as well.
4. I remember we had an issue with Boolean context ( True/False) while passing it to playbooks - saw that you are using the CrowdStrike.FoundDetections as the primary key for that - R u sure it’s going to work with a SIEM as well? 
5. Add CrowdStrike.FoundDetections  to the ‘Malware SIEM Ingestion - Get Incident Data’

And also please check the other comments that I mentioned in the review

File: Packs/CrowdStrikeFalcon/Classifiers/classifier-CrowdStrike_Falcon_Mapper_6.5.json (hunk 1 above)
Comment (David-BMS) [reply Packs/CrowdStrikeFalcon/Classifiers/classifier-CrowdStrike_Falcon_Mapper_6.5.json:668]: There are a million ways to do this
But as the solution works and accepted by CS then why not

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml (hunk 2 above)
Comment (David-BMS) [reply Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml:666]: As you already said
the playbooks is setting it and not the user so not relevant

**Comment by David-BMS** [comment]
@ssokolovich 


> 1. Should move the Incident handling items ( True/False positive classification) in the `CrowdStrike Falcon Malware - Investigation and Response`->  to the right so the manual item will be properly adjusted.
> 2. Why there is hostname info in the context but we don’t have any of that in the layout? Mapper issue?  Same for description and etc (based on the incident that I checked in your Instance)
> 3. CrowdStrike Falcon Malware - Incident Enrichment - should be changed to be shared with context globally? If not, add ExtractedIndicators to the output of this playbook as well.
> 4. I remember we had an issue with Boolean context ( True/False) while passing it to playbooks - saw that you are using the CrowdStrike.FoundDetections as the primary key for that - R u sure it’s going to work with a SIEM as well?
> 5. Add CrowdStrike.FoundDetections  to the ‘Malware SIEM Ingestion - Get Incident Data’
> 
> And also please check the other comments that I mentioned in the review

1. Done
2. Fixed mapper to append device id to also handle incidents, the issue with the description for CS incidents) is caused since you looked at the original mapper and not the updated one.
The valid description looks like
image
Which is very similar to the description in the CS system natively 
3. Done (switched to global)
4. The issue was with using lowercase true/false after we changed it to Upper case it worked fine
See example
image

5.Its already there
image
Line 864

**Review by ssokolovich — APPROVED** [review]

**Comment by David-BMS** [comment]
@julieschwartz18 Can you please review the changes made just to the release notes

**Comment by julieschwartz18** [comment]
Doc review complete for this file. @David-BMS - one question:
Should *Switched CrowdStrike Falcon Malware - Incident Enrichment to use global context.* be under **CrowdStrike Falcon Malware - Investigation and Response**, or should it be under **CrowdStrike Falcon Malware - Incident Enrichment**?

**Pull request merged**